pkg/pillar/scripts/device-steps.sh

```diff
@@ -125,6 +125,7 @@ pidfile = /var/run/xen/xenconsoled.pid
 pidfile = /var/run/xen/xenstored.pid
 pidfile = /var/run/crond.pid
 pidfile = /var/run/logread.pid
+pidfile = /hostfs/run/rsyslogd.pid
```

Comment:
This doesn't have to be prefixed with /hostfs -- please remove

Reply:
Yes, I forgot that we bind mount /run into pillar container. Thanks.